src/main/java/com/minecolonies/colony/ColonyManager.java

```diff
@@ -149,6 +149,34 @@ public static AbstractBuilding getBuilding(@NotNull World w, @NotNull BlockPos p
     }
 
     /**
+     * Get all colonies in this world.
+     *
+     * @param w World
+     * @return a list of colonies
+     */
+    @NotNull
+    public static List<Colony> getColonies(@NotNull World w)
+    {
+        final List<Colony> coloniesInWorld = coloniesByWorld.get(w.provider.getDimension());
+        if (coloniesInWorld == null)
+        {
+            return new ArrayList<>();
+        }
+        return coloniesInWorld;
+    }
+
+    /**
+     * Get all colonies in all worlds.
+     *
+     * @return a list of colonies
+     */
+    @NotNull
+    public static List<Colony> getColonies()
+    {
+        return new ArrayList<>(colonies.values());
```

Comment:
We have to copy anyway, so wrapping that a second time just to deny modification would be too much. And not copying would mean a different collection is backing a list = performance issues (and changes to the keyset change the list) so these should remain as is.